types/serverless/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/serverless",
-    "version": "3.12.9999",
+    "version": "3.13.9999",
```

Comment:
You don't need to do this as versioning is handled by the release bot - I don't think there's any harm, but I'd prefer not doing it to be on the safe side

Reply:
Thank you for your comment. I was not aware that the version number is automatically managed by the bot.
I have fixed it.